pilot5-submission/pilot5-programs/adlbc.r

```diff
@@ -233,8 +234,20 @@ adlbc <- adlb09 %>%
     TRTAN = TRT01AN
   ) %>%
   drop_unspec_vars(adlbc_spec) %>%
+  mutate_if(is.character, ~ replace_na(., "")) %>%
```

Comment:
will this be done for all datasets?

Reply:
we should talk about it the meeting but it think it would solve a lot of issues